elks/arch/i86/kernel/reset-ibm.c

```diff
@@ -0,0 +1,42 @@
+#include <arch/system.h>
+
+/*
+ * This function gets called by the keyboard interrupt handler.
+ * As it's called within an interrupt, it may NOT sync.
+ */
+void ctrl_alt_del(void)
+{
+}
```

Comment:
Can you add hard_reset_now() back in here, it's needed for IBM PC ctrl_alt_del().